### PR TITLE
Adding Statements & Expressions Section

### PR DESCRIPTION
**Pull request by lamarashed**
Statements and Expressions Section

### CODE REVIEW

File: language_reference_manual.md [hunk 1]
```diff
@@ -7,6 +7,139 @@
...
+`"hello,world!";`
+`2;`
+`2.1;`
+`'a';`
+`true;`
+
+### List expressions 
+A list expression can contain zero "empty" or more values; the values are placed in a comma-separated list inside brackets. The values inside the list should have the same data type. 
+**Syntax:**
+`list <str> x = [];`
```
Comment (rk3142) [inline language_reference_manual.md]: Inconsistent with Anne declaration. Use list<str> instead of list <str>

File: language_reference_manual.md [hunk 2]
```diff
@@ -7,6 +7,139 @@
...
+`list <str> x = ["hello", " world "];`
+`list <int> x = [1,2,3];`
+
+### Array expressions 
+An array expression can contain zero "empty" or more values; the values are placed in a comma-separated list inside brackets. The values inside the list should have the same data type with a fixed size.  
+**Syntax:**
+`arr <str> x = [];`
```
Comment (rk3142) [inline language_reference_manual.md]: Use arr<str> instead of arr <str>

File: language_reference_manual.md [hunk 3]
```diff
@@ -7,6 +7,139 @@
...
+`arr <str> x = ["hello", " world "];`
+`arr <int> x = [1,2,3];`
+
+### Dictionary expressions
+A dictionary expression contains a series of keys and values enclosed in curly brackets. A colon is used to separate the key and value, and a comma is used to separate key and value pairs. Keys cannot be duplicated; the duplicated keys will be overwritten. 
+**Syntax:**
+```
+dict <str, str> x =
```
Comment (rk3142) [inline language_reference_manual.md]: Use dict<str, str>

File: language_reference_manual.md [hunk 4]
```diff
@@ -7,6 +7,139 @@
...
+{
+"1":“hello”,
+"2":“world”
+}; 
+```
+### Set expressions
+A set expression is written by enclosing zero or more elements in parentheses; with a comma that separates the elements. 
+The elements inside the list should have the same data type, in no particular order. Sets cannot contain duplicate elements. 
+**Syntax:**
+`set <str> x = ("hello", " world ");`
```
Comment (rk3142) [inline language_reference_manual.md]: Use set<str>

**Review by rk3142 — CHANGES_REQUESTED** [review]

**Comment by lamarashed** [comment]
Addressed reviewer comments

**Review by rk3142 — APPROVED** [review]

**Pull request merged**